packages/app/src/app.ts

```diff
@@ -387,17 +391,30 @@ export class App<Req extends Request = Request, Res extends Response = Response>
 
     let idx = 0
 
-    const loop = () => res.writableEnded || (idx < mw.length && handle(mw[idx++])(req, res, next))
+    const handleNext = (): void => void handle(mw[idx++])(req, res, next)
```

Comment:
Type annotation can be removed.

Reply:
it *can*, but why be implicit when you can be explicit? 

Annotating `void` ensures that the signature is not accidentally changed in future.